etc/proto/Dockerfile

```diff
@@ -4,12 +4,16 @@ LABEL maintainer="msteffen@pachyderm.io"
 RUN apt-get update -yq
 RUN apt-get install -yq unzip
 
-RUN wget https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-linux-x86_64.zip -O protoc.zip
+RUN wget https://github.com/google/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip -O protoc.zip
 RUN unzip protoc.zip -d /
 RUN cp -r /include /bin
 
 RUN go get -u -v github.com/golang/protobuf/proto
 RUN go get -u -v github.com/gogo/protobuf/protoc-gen-gofast
+# RUN go get -d -u -v github.com/gogo/protobuf/protoc-gen-gofast \
+#  && cd ${GOPATH}/src/github.com/gogo/protobuf/protoc-gen-gofast \
+#  && git checkout origin/dev \
+#  && go install ./...
```

Comment:
Why are we adding these lines but commented out?

Reply:
This is a mistake. I added these while debugging the `"golang.org/x/net/context"` vs `"context"` issue (https://github.com/pachyderm/pachyderm/issues/3265) and forgot to remove them. They're gone now

Reply:
Makes sense, can't approve to confirm because I already approved.